starboard/tools/api_leak_detector/api_leak_detector.py

```diff
@@ -104,6 +104,9 @@
     'munmap',
     'mprotect',
     'msync',
+    'stat',
+    '__stat_time64',
```

Comment:
Those 2 symbols `__stat_time64` and `__xstat` look like implementation details from one of the platforms.
Do you get them on Evergreen or the stub?
We should not list them here. If those are only from the stub we should add them to the manifest files. 
https://source.corp.google.com/h/lbshell-internal/cobalt_src/+/COBALT:starboard/tools/api_leak_detector/stub/

For Evergreen those show up we'll have to fix them.